apps/src/turtle/turtle.js

```diff
@@ -1585,7 +1585,7 @@ Artist.prototype.step = function (command, values, options) {
       this.visualization.ctxScratch.save();
       this.visualization.ctxScratch.translate(this.visualization.x, this.visualization.y);
       this.visualization.ctxScratch.rotate(this.visualization.degreesToRadians_(this.visualization.heading));
-      this.visualization.ctxScratch.drawImage(img, 0, 0, width, height, -width / 2, -height, width, height);
+      this.visualization.ctxScratch.drawImage(img, -width / 2, -height, width, height);
```

Comment:
I repro the `IndexSizeError: DOM Exception 1` with the following code:

```
phantomjs> img = new Image(100, 100);
phantomjs> ctx = document.createElement('canvas').getContext('2d');
phantomjs> ctx.drawImage(img, 0, 0, 100, 100);
IndexSizeError: DOM Exception 1

  phantomjs://repl-input:1 in global code
```

Can we continue to pass all 9 arguments here to fix the failing test?